src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -99,31 +106,49 @@ object Arisa : ConfigSpec() {
-                        "[Mojang Support Center|http://help.mojang.com/customer/portal/articles/364794-where-can-i-find-more-help-and-technical-support-]."
+                "*Thank you for your report!*\n" +
+                        "We're actually already tracking this issue in *{DUPLICATE}*, so I've resolved and linked this ticket as a duplicate.\n" +
+                        "\n" +
+                        "If you would like to add a vote and any extra information to the main ticket it would be appreciated.\n" +
+                        "\n" +
+                        "If you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=MC] to see if the issue has already been mentioned.\n" +
+                        "\n" +
+                        "*Quick Links*:\n" +
+                        "\uD83D\uDCD3 [Issue Guidelines|https://bugs.mojang.com/projects/MC/summary] -- \uD83D\uDCAC [Community Support|https://discord.gg/58Sxm23] -- \uD83D\uDCE7 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- \uD83D\uDCD6 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]"
             )
             val moddedMessage by optional(
-                "This ticket is _invalid_ as it relates to a modified or third-party client, server, or launcher.\r\n" +
-                        "* Any non-standard client/server/launcher build needs to be taken up with the appropriate team, not Mojang.\r\n" +
-                        "* Any plugin issues need to be addressed to the creator of the plugin or resource pack.\r\n" +
-                        "* This site is for addressing issues related to the *base unmodded Minecraft*; " +
-                        "any modded system _invalidates_ your ticket, unless the behavior can be reproduced without mods.\r\n* " +
-                        "Additionally, if you have problems on large-scale modded servers, please report it to their site. It's probably not a bug in Minecraft."
-            )
+                "*Thank you for your report!*\n" +
+                        "However, this issue is {color:#FF5722}*Invalid*{color}.\n" +
+                        "\n" +
+                        "Your game, launcher or server is modified.\n" +
+                        "If you can reproduce the issue in a vanilla environment, please recreate the issue.\n" +
+                        "\n" +
+                        "* Any non-standard client/server/launcher build needs to be taken up with the appropriate team, not Mojang.\n" +
+                        "* Any plugin issues need to be addressed to the creator of the plugin or resource pack.\n" +
+                        "* If you have problems on large servers, such as The Hive and Hypixel, please contact them first as they run modified server software.\n" +
+                        "\n" +
+                        "\n" +
+                        "*Quick Links*:
```

Comment:
You have an extra new line here and thats breaking the text below

Reply:
Should be fixed

Reply:
Did you push? No new commit here.

Reply:
Not yet, hang on

Reply:
Should be done now?

Reply:
Ah, not the `"\n"`
You have a new line character after quick links. The line needs to be changed to either `"*Quick Links*:\n" +` or line 43 needs to be moved into the same line.

The \n you reomved was okay.

Reply:
Oh! I'll put that back and fix it, I'll have to stop for school/lunch though so it'll be a while before I've fixed all of the reviewed things

Reply:
Check now?

Reply:
The new line should be good now, the build fails for different reasons now 😅